tools/Colibri Core (FoLiA XML document).json

```diff
@@ -25,16 +26,11 @@
         "minlength": 1,
         "maxlength": 5,
         "cooc": 0,
-        "npmi": -1.0
+        "npmi": -1
```

Comment:
Can you say something about these changes as well (change of precision in some tools' manual parameters)? Were these tools broken or working?

Reply:
This change was made by `jq` (json formatting and manipulation tool), I used it to automatically add the id and remove the output. There is no semantic difference between -1.0 and -1 in json/javascript because all numbers have floating point representation.